pkg/cluster/generateconfig.go

```diff
@@ -28,143 +24,67 @@ import (
 	"github.com/openshift/installer/pkg/types/validation"
 	"golang.org/x/crypto/ssh"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/Azure/ARO-RP/pkg/api"
-	"github.com/Azure/ARO-RP/pkg/bootstraplogging"
-	"github.com/Azure/ARO-RP/pkg/cluster"
-	"github.com/Azure/ARO-RP/pkg/util/azureerrors"
-	"github.com/Azure/ARO-RP/pkg/util/deployment"
+	"github.com/Azure/ARO-RP/pkg/api/validate"
 	"github.com/Azure/ARO-RP/pkg/util/pullsecret"
 	"github.com/Azure/ARO-RP/pkg/util/stringutils"
 	"github.com/Azure/ARO-RP/pkg/util/subnet"
 	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
-func (m *manager) Create(ctx context.Context) error {
-	var err error
+func (m *manager) checkQuotas(ctx context.Context) error {
+	// we don't re-call Dynamic on subsequent entries here.  One reason is
+	// that we would re-check quota *after* we had deployed our VMs, and
+	// could fail with a false positive.
```

Comment:
I think the PR breaks this.  If the RP crashes after deploy starts and the process restarts on another backend, quota will be checked again and may fail the second time around.

Reply:
I think I've fixed this.